src/main/java/de/tum/cit/aet/artemis/core/dto/PasskeyDto.java

```diff
@@ -0,0 +1,11 @@
+package de.tum.cit.aet.artemis.core.dto;
+
+import java.time.Instant;
+
+/**
+ * DTO for passkey information.
+ *
+ * @param credentialId encoded as base64url
+ */
+public record PasskeyDto(String credentialId, String label, Instant created, Instant lastUsed) {
```

Comment:
I also just realized, iirc DTOs are usually in capital letters in Artemis.

```suggestion
public record PasskeyDTO(String credentialId, String label, Instant created, Instant lastUsed) {
```